ERCS/erc-7786.md

```diff
@@ -181,6 +181,15 @@ A gateway MAY operate in both active and passive modes, or it MAY switch from op
 
 A receiver SHOULD support both active and passive modes for any gateway. This is accomplished by first checking whether the caller of `receiveMessage` is a known gateway, and only validating the message if it is not; the first case supports an active mode gateway, while the second case supports a passive mode gateway.
 
+### Properties
+
+The protocol underlying a pair of gateways is expected to guarantee a series of properties. For detailed definition and discussion we refer to XChain Research’s [Cross-chain Interoperability Report](https://github.com/0xFableOrg/xchain/blob/7789933bba24e2dd893cf515157af70474e7180b/README.md).
+
+- The protocol MUST guarantee Safety: A message is delivered at the destination if and only if it was sent at the source, and the sending transaction was finalized.
+- The protocol MUST guarantee Liveness: A sent message is delivered at the destination eventually, assuming Liveness and censorship-resistance of the source and destination chains.
+- The protocol SHOULD guarantee Timeliness: A sent message is delivered at the destination within a bounded delivery time, which should be documented.
+- The above properties SHOULD NOT rely on trust in some centralized actor.
+
```

Comment:
Can we add a sentence about only once execution here? I think this is an important property.

Reply:
IMO that is part of Safety. I would expend the safety part to mention that. WDYT?